app/src/main/AndroidManifest.xml

```diff
@@ -24,7 +24,7 @@
     <uses-feature android:name="android.hardware.camera" />
     <uses-feature android:name="android.hardware.microphone" />
     <uses-feature android:name="android.hardware.bluetooth_le" android:required="false"/>
-    
+
```

Comment:
Be careful with adding not necessary diffs

Reply:
ok